connector/src/main/java/org/geysermc/connector/configuration/GeyserConfiguration.java

```diff
@@ -104,6 +106,12 @@
         String getMotd2();
 
         String getServerName();
+
+        boolean isEnableProxyProtocol();
+
+        List<String> getProxyProtocolWhitelistedIPs();
+
+        List<CIDRMatcher> getWhitelistedIPsMatchers();
```

Comment:
Since this isn't strictly a config value: add a Javadoc here explaining what this does?